src/main/java/org/breedinginsight/brapps/importer/model/imports/experimentObservation/ExperimentObservation.java

```diff
@@ -290,6 +290,9 @@ public BrAPIObservationUnit constructBrAPIObservationUnit(
             BrAPIObservationTreatment treatment = new BrAPIObservationTreatment();
             treatment.setFactor(getTreatmentFactors());
             observationUnit.setTreatments(List.of(treatment));
+
+            // Store treatments in additionalinfo as well for temporary BreedBase workaround
+            observationUnit.putAdditionalInfoItem(BrAPIAdditionalInfoFields.TREATMENTS, List.of(treatment));
```

Comment:
This may be better to do in the BrAPIObservationUnitDAO to encapsulate that logic only there.